src/js/views/views/LeagueDashboard.js

```diff
@@ -116,6 +120,7 @@ const LeagueDashboard = ({abbrev, ast, astRank, att, cash, completed = [], confT
                         :
                             <div>
                                 <h3>Upcoming Games</h3>
+                                <div>{gamesRemaining} games remaining ({(percentComplete * 100).toFixed(1)}% complete)</div>
```

Comment:
Change <div> to <p>, maybe? I don't like how it's so close to the games list, looks strange. Besides that, very nice!